nodewatcher/modules/devices/ubnt/unifi.py

```diff
@@ -84,6 +84,15 @@ class UBNTUniFiAPACLite(cgm_devices.DeviceBase):
     }
 
 
+class UBNTUniFiACMesh(UBNTUniFiAPACLite):
+    """
+    UBNT UniFi AC Mesh device descriptor.
+    """
+
+    identifier = 'ub-umesh-ac'
```

Comment:
I think it should be `ub-uap-ac-mesh`. Because it is still part of `UniFi AP AC` family: https://store.ubnt.com/products/unifi-ac-mesh-ap